migrations/repotracker_cleanup.go

```diff
@@ -0,0 +1,452 @@
+package migrations
+
+import (
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/build"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/model/version"
+	"github.com/mongodb/anser"
+	"github.com/mongodb/anser/db"
+	"github.com/mongodb/anser/model"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
you can use anser's db.Document{} rather than bson.M, but I think that you still end up needing to depend on bson for RawDoc, so it doesn't really matter.